### DISCUSSION TITLE
Wiki suggestions

### DISCUSSION THREAD

**Discussion by ballzzpta** [dennisrijsdijk/firebot-streamdeck#4] [Q&A and Support] [answered]
Not a question, more of feedback for the wiki. 
When installing a new "instance" for a multi pc setup, I found that the streamdeck window needed to be closed and re-opened in order to see the commands for the instance I created.
However this is not conveyed in the wiki, some users might think this doesn't work.

**Comment by dennisrijsdijk** [comment, marked as answer]
I believe this is related to #2 rather than being a direct behavior of the plugin.

I'll update the issue to reflect this, but I'll leave this discussion open for now. Once that issue is fixed, we can re-evaluate to see if this was related to that issue, or if it's seperate. This is definitely not intended behavior, which is why it wasn't mentioned in the wiki.

Thanks for letting me know!
